routes/baseDBMethods.js

```diff
@@ -20,14 +22,42 @@ function handleDocResponse(ctx, doc) {
   }
 }
 
+function processPaginationParameters(ctx) {
+  const pageNumber = parseInt(ctx.query.page, 10) || DEFAULT_PAGE_NUMBER;
+  if (pageNumber < 1) {
+    throw new RangeError('Query parameter \'page\' must be a positive number');
+  }
+
```

Comment:
Why pageNumber is const and pageSize is let?
Must be ESlint messing with us again

Reply:
Because `pageNumber` is not changed later, and `pageSize` is (`pageSize` is limited by `MAX_ALLOWED_PAGE_SIZE`).